Sources/Sentry/Profiling/SentryProfiledTracerConcurrency.mm

```diff
@@ -80,17 +80,13 @@
 void
 _unsafe_cleanUpContinuousProfilerV2()
 {
-    _gInFlightRootSpans -= 1;
-
-    if (_gInFlightRootSpans < 0) {
-#    if SENTRY_TEST || SENTRY_TEST_CI
-        SENTRY_CASSERT(NO, @"Attempted to decrement count of root spans to less than zero.");
-#    else
-        SENTRY_LOG_DEBUG(@"Attempted to decrement count of root spans to less than zero.");
-#    endif // SENTRY_TEST || SENTRY_TEST_CI
+    if (_gInFlightRootSpans == 0) {
+        SENTRY_TEST_FATAL(@"Attempted to decrement count of root spans to less than zero.");
+    } else {
+        _gInFlightRootSpans -= 1;
```

Comment:
This was a bad part, since _gInFlightRootSpans is an unsigned int, decrementing it from 0 would underflow it to UINT_MAX